snorkel/labeling/model/label_model.py

```diff
@@ -794,10 +768,10 @@ def _break_col_permutation_symmetry(self) -> None:
             2. diag(O) = sum(mu @ P, axis=1)
         Then any column permutation matrix Z that commutes with P will also equivalently
         satisfy these objectives, and thus is an equally valid (symmetric) solution.
-        Therefore, we select the solution where the most LFs are estimated to be more
-        accurate than not when not abstaining, i.e., where for the majority of LFs,
+        Therefore, we select the solution that maximizes the summed probability of the
+        LFs being accurate when not abstaining.
 
-            P(\lf = Y) > P(\lf != Y, \lf != -1).
+            \sum_lf \sum_{y=1}^{cardinality} P(\lf = y, Y = y)
 
         This is the standard assumption we have made in algorithmic and theoretical
```

Comment:
Update comment? This is no longer the standard assumption we have made to date.

Reply:
We could just delete this comment.  I guess we've often made a slightly different assumption, namely that more LFs are > 50% than not (phrased in the binary, single LF param setting), rather than that the sum of their accuracies is maximized... but not sure worth confusing people here?  Just delete this bit?